apps/admin/backend/src/store.ts

```diff
@@ -329,6 +375,159 @@ export class Store {
     );
   }
 
+  /**
+   * Adds link record representing the association between a ballot style and a precinct.
+   */
+  private createBallotStylePrecinctLinkRecord({
+    electionId,
+    ballotStyleId,
+    precinctId,
+  }: {
+    electionId: Id;
+    ballotStyleId: BallotStyleId;
+    precinctId: PrecinctId;
+  }): void {
+    this.client.run(
+      `
+        insert into ballot_styles_to_precincts (
+          election_id,
+          ballot_style_id,
+          precinct_id
+        ) values (
+          ?, ?, ?
+        )
+      `,
+      electionId,
+      ballotStyleId,
+      precinctId
+    );
+  }
+
+  /**
+   * Adds link record representing the association between a ballot style and a precinct.
+   */
+  private addVotingMethodRecords({ electionId }: { electionId: Id }): void {
+    const params: Bindable[] = [];
+    for (const votingMethod of Tabulation.SUPPORTED_VOTING_METHODS) {
+      params.push(electionId, votingMethod);
+    }
+
+    this.client.run(
+      `
+          insert into voting_methods (
+            election_id,
+            voting_method
+          ) 
+          values 
+            ${Tabulation.SUPPORTED_VOTING_METHODS.map(() => '(?, ?)').join(
+              ',\n'
+            )}
+        `,
+      ...params
+    );
+  }
```

Comment:
These records exist simply to be able to get the voting method "cartesian product" with other splits through the DB rather than adding logic outside of DB.

Reply:
Could they be added after the `create table` in `schema.sql`?

Reply:
Definitely. My thinking here, though, was that eventually this may be a jurisdiction by jurisdiction setting so I didn't want it at the `schema` level.